tests/mocks/webtaskStorage.js

```diff
@@ -1,21 +1,31 @@
 const _ = require('lodash');
 
-module.exports = function(data, onDataChanged) {
+module.exports = function(data, onDataChanged, beforeDataChanged) {
   var webtaskData = data;
   return {
     get: function(cb) {
       if (data && data.name === 'Error') {
         return cb(data);
       }
-      return cb(null, webtaskData);
+      return cb(null, _.cloneDeep(webtaskData));
```

Comment:
@sandrinodimattia: This was done to prevent clients from changing the persisted mock data by having a direct reference to the `webtaskData` object. Now the only way to save a change is to call `set`, which is how the real webtask storage works anyway.